Build issues

1) basename: invalid option -- 'a'

```
C(SSE2)   obj/arch/sse2_c.o
C(AVX2)   obj/arch/avx2_c.o
ASM       obj/apps/example/asm_dasc.o
basename: invalid option -- 'a'
Try `basename --help' for more information.
make[1]: *** [program/programs.inc] Error 123
```

2) For some reason cloning http://github.com/SnabbCo/luajit.git fails. I had to replace that with http2:// in .gitmodules, deps/pflua/.gitmodules,

--- a/src/Makefile
+++ b/src/Makefile
@@ -191,7 +191,7 @@ $(INCOBJ): obj/%_inc.o: %.inc Makefile program/programs.inc | $(OBJDIR)
 
 # Create list of programs that exist
 program/programs.inc:
-	@ls -1d program/*/ | xargs basename -a > $@
+	@(for d in program/*/; do basename $$d; done) > $@
 
 .FORCE: program/programs.inc
 

